root/layout/components/Header.js

```diff
@@ -9,20 +9,41 @@
 
 import React from 'react';
 
+import {withCatalystContext} from '../../context';
+import {hydrateClient} from '../../utility/hydrate';
+
 import HeaderLogo from './HeaderLogo';
 import TopMenu from './TopMenu';
 import BottomMenu from './BottomMenu';
 
-const Header = () => (
-  <div className="header">
-    <a className="logo" href="/" title="MusicBrainz">
+const HeaderContents = ({$c}) => (
+  <>
+    <a className="navbar-brand my-auto" href="/" title="MusicBrainz">
       <HeaderLogo />
     </a>
-    <div className="right">
-      <TopMenu />
-      <BottomMenu />
+    <div className="collapse navbar-collapse flex-md-column">
```

Comment:
I was expecting the menus to collapse into one of those clickable bootstrap menus when on phone version, but it does seem to just put them one after another in different lines and nothing else:

![Screenshot from 2019-11-16 01-32-10](https://user-images.githubusercontent.com/1069224/68982857-4139e180-0811-11ea-9a27-9db612ed64a9.png)

(FWIW I'd probably expect to collapse all into one menu when on small screens, and leave the search shown on its own)